### DISCUSSION TITLE
EFB performance

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#3283] [Q&A] [answered]
first of all, i´m so impressed of all the new and upcoming features! 
especially the EFB.

I saw in many posts and videos, that there will be an implementation of a map. My fear is that it costs frames and performance of the overall experience.

will be there any option to en-/disable the efb functions, like the live map? Or is there any analysis of the impact in fps/performance?

THANK YOU for all your hard work!

**Comment by sidnov** [comment, marked as answer]
Hi there! Since the EFB is at a very initial phase, not a lot of features have been implemented. But the frame-intensive features like live map are up and running, and the best part is there is literally no performance impact at all.

Happy flying,
Sid
QA Tester

**Reply by andrejetcat** [reply]
thank you for your answer! Thats good news! 


Thank you for all your hard work! I´m very looking forward to the new features! :)

**Reply by peterwirtzgericke** [reply]
Hey,

since Update of today (dev) I have performance probs, lagg every 2 seconds.

**Reply by andrejetcat** [reply]
...that was my fear :(

**Reply by hotshotp** [reply]
> Hey,
> 
> since Update of today (dev) I have performance probs, lagg every 2 seconds.

Not related to the efb, but the rmp
